Update v20 packaging for Xcode 26

tgoyne opened the pull request.

tgoyne committed:
Fix test compilation with Xcode 26

tgoyne committed:
Skip codesigning release binaries

tgoyne committed:
Upgrade gem versions used for building docs

tgoyne committed:
Package for Xcode 26.0.1 and drop Xcode < 16.3

tgoyne committed:
Don't try to publish docs

tgoyne committed:
Remove dead links from podspecs

tgoyne committed:
Remove slack notification for release completing

LucasAdam commented:
@tgoyne are you still planing on getting this released? Or are you waiting for 26.1? 🙏🏼

ilendemli commented:
Any news @tgoyne?

tgoyne committed:
Run on macos-15

tgoyne committed:
Add Xcode 26.1

tgoyne committed:
Explicitly install the required platform for CI jobs

tgoyne committed:
Run on macos-15-arm64

rahulchavan06 commented:
Is there any tentative timeline to provide XCode 26 support?

tgoyne commented:
I keep running into annoying-to-debug issues with the github actions macos env.

tgoyne committed:
Update a bunch of Xcode and macOS versions

tgoyne committed:
Fix a new swiftlint failure

tgoyne committed:
Work around XCTAssertEqual no longer working on bitfields

tgoyne committed:
Use iPhone 17 for Xcode 26 testing

tgoyne committed:
Install platform support for release builds

tgoyne committed:
Build core with Xcode 26

tgoyne committed:
Update to core 20.1.3

tgoyne committed:
Disable fail-fast for core builds

Individual jobs often have transient failures due to GHA issues and fail-fast
ensures we cannot make forward progress.
